src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudWindowsIT.java

```diff
@@ -0,0 +1,299 @@
+package com.google.jenkins.plugins.computeengine;
+
+import com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey;
+import com.cloudbees.plugins.credentials.*;
+import com.cloudbees.plugins.credentials.common.StandardUsernameCredentials;
+import com.cloudbees.plugins.credentials.domains.Domain;
+import com.cloudbees.plugins.credentials.domains.DomainRequirement;
+import com.google.api.services.compute.model.Image;
+import com.google.api.services.compute.model.Instance;
+import com.google.api.services.compute.model.Operation;
+import com.google.jenkins.plugins.computeengine.client.ClientFactory;
+import com.google.jenkins.plugins.computeengine.client.ComputeClient;
+import com.google.jenkins.plugins.computeengine.ssh.GoogleKeyPair;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials;
+import com.google.jenkins.plugins.credentials.oauth.ServiceAccountConfig;
+import hudson.model.Node;
+import hudson.model.labels.LabelAtom;
+import hudson.slaves.NodeProvisioner;
+import org.junit.*;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.*;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+import java.util.logging.StreamHandler;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ComputeEngineCloudWindowsIT {
+    private static Logger log = Logger.getLogger(ComputeEngineCloudWindowsIT.class.getName());
+
+    private static final String CLOUD_NAME = "integration";
+    private static final String NAME_PREFIX = "integration";
+    private static final String REGION = format("projects/%s/regions/us-west1");
```

Comment:
The format method isn't necessary if you move the initialization of these fields into the init() method where you're already checking for the proper Env vars. Best not to repeat the same logic. You can change the ones which currently use the format method from: "private static final" ->  "private final".